ShowBuildMenu.sh

```diff
@@ -14,8 +15,10 @@ async_generator_path=""
 
 if [ ! -f $BUILD_TOOL_PATH ]
 then
-	dotnet build ./Tools/BuildTool/BuildTool.sln -c Release -o bin
+	cd ./Tools/BuildTool
+	dotnet build BuildTool.sln -c Release -o bin 
 fi
+cd "$SCRIPT_PATH"
```

Comment:
I would just have used `cd ../..` before the `fi`, but why not.

Reply:
I was thinking, but decided against it.